CMakeLists.txt

```diff
@@ -132,10 +132,18 @@ if(LINUX)
   set(HAVE_UDEV ${UDEV_FOUND})
   find_package(blkid REQUIRED)
   set(HAVE_BLKID ${BLKID_FOUND})
+  if(WITH_LIBCEPHFS OR WITH_KRBD)
+    # keyutils is only used when talking to the Linux Kernel key store 
+    find_package(keyutils REQUIRED)
+    set(HAVE_KEYUTILS ${KEYUTILS_FOUND})
+  else()
+    set(HAVE_KEYUTILS OFF)
```

Comment:
i'd suggest just drop the `else()` branch. as an unset value is evaluated to FALSE in cmake.